doc/en/user/source/community/jwt-headers/configuration.rst

```diff
@@ -0,0 +1,211 @@
+.. _community_jwtheaders_config:
+
+JWT Headers configuration
+=========================
+
+The JWT Headers module covers three main use cases;
+
+#. Simple Text, JSON, or JWT headers for the username
```

Comment:
The module is called JWT though, and then it supports also generic JSON... maybe it should be given a more general name?

Reply:
True - if you have a better name, let me know what it is.  I started with JWT Headers and couldn't think of a simple name that would be more descriptive. 

JWT does mean "JSON Web Token".

Reply:
Perhaps "JSON token" module? 

JWT is quite specific, it's a well-known specification. 
One would not go and look into it if they knew their JSON payload was not compliant with JWT. 

But with "JSON token" we have a chance of grabbing attention of both JWKT and random JSON users. Maybe. Just thinking out loud here :rofl:

Reply:
I agree with you - however, a lot of the work is in the validation of the JWT.  Without that, I probably would have just upgraded the Headers extension (and added a new Role Source + Role Converter).

The other reason for this module was to make the OIDC (and OAUTH2 robot access), thats in the oauth2-* community modules, much easier to configure, maintain, and test.  As well as some extra functionality.

So, using "JWT" seemed appropriate - since it will grab the attention of anyone using OIDC or OAUTH2 (i.e. Apache mod_auth_openid users).